src/components/theme/Themes.js

```diff
@@ -1,7 +1,6 @@
-// import backgroundImageLight from '../../images/background_image_yearn_watch_light.svg';
 import backgroundImageDark from '../../images/background_image_yearn_watch_dark.jpeg';
-import backgroundImageLightNoMitch from '../../images/yearn_background_light_no_mitch.svg';
-import backgroundLoading from '../../images/loadingBackground1.svg';
+import backgroundImageLightNoMitch from '../../images/yearn_background_light_no_mitch.png';
```

Comment:
PNG files of 2MB are too big for web, they need to be optimized

Reply:
Deleted image png